rfcs/070-concepts-api-changes/README.md

```diff
@@ -0,0 +1,324 @@
+and a `workType` filter (including the ID of the 'Books' `workType`).
+
+### Subject theme pages
+
+New subject theme page designs include additional tabs listing works linked to sub topics (child themes) of the given theme:
+
+> <img src="img/works_tabs_concept.png" alt="works_tabs_concept" width="600"/>
+
+To retrieve these works from the catalogue API, we need the *single concept endpoint* to return a list of relevant narrower topics
+(in addition to the fields listed in the tables above):
+
+| Field name  | Details                                                                                                                                                                                                                                          |
+|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| broaderThan | Stored in the graph as SourceConceptNarrowerThan (edge direction reversed) and SourceConceptHasParent (edge direction reversed) edges. |
+
+We can then make separate API calls to the catalogue API to retrieve a list of works for each sub topic.
+
+However, this poses a challenge, because the relationships between works and concepts are not inherited. For example, if a work references
+the concept `Sanitation - history`, it does **not** automatically reference the concept `Sanitation`, even though
+`Sanitation - history` is a child (sub topic) of `Sanitation`. In general, this means that works which reference a child of
+a given theme (but do not reference the theme directly) will not be automatically included in the `All` tab in
+the screenshot above.
+
+We can address this by modifying the catalogue API call to filter for all works referencing a given theme **or** any of its
+sub topics (i.e. retrieving all works referencing either `Sanitation` or `Sanitation - history`). However, this would become complicated
+and computationally expensive if done across multiple levels (i.e. retrieving works referencing sub topics, works referencing
+sub topics of sub topics, etc.). Therefore, we should limit ourselves to one level only (i.e. direct children).
+
+
+## Merging concepts across sources
+
+Due to storing `HAS_SOURCE_CONCEPT` edges between catalogue concepts and source concepts, and `SAME_AS` edges
+between source concepts, the catalogue graph allows us to 'merge' identical concepts into a single unified theme page.
+For example, if a work references a LoC subject heading `Sanitation` and a different work references a MeSH term `Sanitation`,
+both of these works would link to (and be listed on) the same `Sanitation` theme page.
+
+To enable this functionality, all 'merged' concepts will be stored in the Elasticsearch index as a single document,
+which will include all of their canonical IDs. This is similar to the `sameAs` field
```

Comment:
It may be preferable to keep seperate elasticsearch documents for each concept ID; doing so would allow us to deal with the lowered reliability of "correct" relationships to other works through concepts inherent with label-derived concepts. 

Put another way: If you arrive at a label derived concept page, the actual meaning is necessarily ambiguous so it makes sense the works on that page might have a broader / less accurate semantic relationship. However if you arrive at a MeSH derived concept page, the meaning is quite specific and the connection to other works through a label-derived concept match is less useful.

Reply:
We may want to implement a difference in the way these pages are constructed, either through how attributes are merged from a related concept onto the indexed document, or via how queries for works and images based on matched concepts in these pages are weighted.

For example on an explicitly matched (NOT label derived) "LoC" concept page, we may want to reduce the visibility of works that are matched through a label-derived concept. You can imagine writing an elasticsearch query which weights matches to works with label derived concepts as less relevant than matches through concepts in a known ontology.